packages/sdk/schema/src/common/message.ts

```diff
@@ -258,6 +258,23 @@ export namespace ContentBlock {
   }).pipe(Schema.mutable);
   export interface Toolkit extends Schema.Schema.Type<typeof Toolkit> {}
 
+  export const ConversationFork = Schema.TaggedStruct('threadFork', {
```

Comment:
Since these are namespaced i think fork/merge is enough, but otherwise Fork/MergeThread? (or at least agree with the tagged string)